hypertrace-gradle-publish-maven-central-plugin/README.md

```diff
@@ -0,0 +1,66 @@
+# Hypertrace Publish Maven Central Plugin
+
+###### org.hypertrace.publish-maven-central-plugin
+
+### Purpose
+This plugin configures the target project to publish its java artifacts to maven central. 
+It uses the `maven-publish` and `signing` plugins internally to do this.
+By default, it will publish to https://s01.oss.sonatype.org/service/local/staging/deploy/maven2/. 
+It also adds `javadocJar` and `sourcesJar` tasks to the project.
+The following credentials must be provided as gradle properties in the default configuration:
+- ossrhUsername
+- ossrhPassword
+
+For signing artifacts, following properties must be provided as gradle properties:
+- signingKey
+- signingPassword
+
+Each property described below can be configured in the DSL. The default values are shown for each property,
+all of which, with the exception of license, can be omitted if left unchanged.
+```kotlin
+ hypertracePublishMavenCentral {
+    url // The URL for the publication represented by the POM.
+    scmConnection // The connection URL of the SCM.
+    scmDeveloperConnection // The developer connection URL of the SCM
+    scmUrl // The browsable repository URL of the SCM
+    licenses // Configures the licenses for the publication represented by the POM
+    developers // Configures the developers for the publication represented by the POM
```

Comment:
Is this necessary? Not sure what the maven central requirements are - can it be like hypertrace-dev@hypertrace.org (or some valid email), so it's not an individual?

Reply:
updated the default values with community@hypertrace.org.